doc/releases/changelog-0.40.0.md

```diff
@@ -69,47 +223,308 @@
     [(#6666)](https://github.com/PennyLaneAI/pennylane/pull/6666)
-  * Implemented helper functions for calculating one-mode PES, two-mode PES, and
-    three-mode PES.
-    [(#6616)](https://github.com/PennyLaneAI/pennylane/pull/6616)
-    [(#6676)](https://github.com/PennyLaneAI/pennylane/pull/6676)
-  * Implemented wrapper function for vibrational Hamiltonian calculation and dataclass
-    for storing the data.
-    [(#6652)](https://github.com/PennyLaneAI/pennylane/pull/6652)
-  * Implemented functions for generating the vibrational Hamiltonian in VSCF basis
-    [(#6688)](https://github.com/PennyLaneAI/pennylane/pull/6688)
-
-* Added support to build a vibrational Hamiltonian in Taylor form.
-  [(#6523)](https://github.com/PennyLaneAI/pennylane/pull/6523)
+
+  * Separate normal mode frequencies and localize them with :func:`~.qchem.localize_normal_modes`.
+    [(#6453)](https://github.com/PennyLaneAI/pennylane/pull/6453)
+
+<h3>Labs: a place for unified and rapid prototyping of research software 🧑‍🔬</h3>
+
+The new :mod:`qml.labs <pennylane.labs>` module will house experimental research software 🔬. Features here may be useful
+for state-of-the-art research, beta testing, or getting a sneak peek into *potential* new features before
+they are added to PennyLane.
+
+The experimental nature of this module means features may not integrate well with other 
+PennyLane staples like differentiability, JAX, or JIT compatibility. There may also be unexpected
+sharp bits 🔪 and errors ❌.
+
+.. warning:: 
+    This module is **experimental**! Breaking changes and removals will happen without warning.
+    Please use these features carefully and let us know your thoughts. Your feedback will inform
+    how these features become a part of mainline PennyLane.
+
+<h4>Resource estimation</h4>
+
+* Resource estimation functionality in Labs is focused on being light-weight and flexible. 
+The Labs :mod:`qml.labs.resource_estimation <pennylane.labs.resource_estimation>` module involves modifications to core PennyLane that reduce the
+memory requirements and computational time of resource estimation. These include new or modified
+base classes and one new function:
+  * :class:`~.labs.resource_estimation.Resources` - This class is simplified in `labs`, removing the arguments: `gate_sizes`, `depth`,
+  and `shots`. [(#6428)](https://github.com/PennyLaneAI/pennylane/pull/6428)
+  * :class:`~.labs.resource_estimation.ResourceOperator` - Replaces :class:`~.resource.ResourceOperation`, expanded to include decompositions. [(#6428)](https://github.com/PennyLaneAI/pennylane/pull/6428)
+  * :class:`~.labs.resource_estimation.CompressedResourceOp` - A new class with the minimum information to estimate resources:
+  the operator type and the parameters needed to decompose it. [(#6428)](https://github.com/PennyLaneAI/pennylane/pull/6428)
+  * :class:`~.labs.resource_estimation.ResourceOperator` versions of many existing PennyLane operations, like Pauli operators,
```

Comment:
```suggestion
  * :class:`~.labs.resource_estimation.ResourceOperator` - versions of many existing PennyLane operations, like Pauli operators,
```

Reply:
:thinking: 
Not super important. But I like it without the hyphen.